pyproject.toml

```diff
@@ -25,32 +26,33 @@ classifiers = [
     "Operating System :: Unix",
     "Operating System :: MacOS",
 ]
-requires-python = ">=3.9"
-dependencies = [
-    "aws-cdk-lib==2.48.0",
-    "aws-cdk.aws-lambda-python-alpha",
-    "constructs>=10.0.0,<11.0.0",
-    "boto3",
-    "opensearch-py",
-    "requests",
-    "python-jose"
-]
 
-[project.optional-dependencies]
-dev = [
-    "pytest==6.2.5",
-    "moto"
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+# Required for aws-cdk.aws-lambda-python-alpha
+aws-cdk-lib = ">=2.66.0,<3.0.0"
+#aws-cdk-lib = "2.48.0"
+boto3 = "^1.26.78"
+"aws-cdk.aws-lambda-python-alpha" = "^2.66.0a0"
+constructs = ">=10.0.0,<11.0.0"
+black = "^23.1.0"
```

Comment:
I'd add this and ruff to a `dev` or `all` dependency section, which can also depend on the tests dependency below then.